HelpSource/Classes/AbstractGridLines.schelp

```diff
@@ -35,100 +36,105 @@ CLASSMETHODS::
 METHOD:: new
 
 argument:: spec
-The ControlSpec that defines the minimum and maximum values, warp and step.
+A link::Classes/ControlSpec:: that defines the numerical range, warp, and step.
 
-returns:: a AbstractGridLines
+returns:: The newly instantiated object.
```

Comment:
Because all of the method descriptions of `AbstractGridLines` are inherited by its subclasses via `copymethod::`, I was trying to make it generic enough to apply to the subclasses as well.
I've changed it to "An instance of this class."

But if it's preferred to state the class, I can just add "An instance of xxx." in each of the subclasses.